### DISCUSSION TITLE
Cannot use with SKR2 mainboard.  Error "pin PA2 is reserved for t5uid1"

### DISCUSSION THREAD

**Discussion by phoenixfire425** [Desuuuu/klipper#65] [Q&A] [answered]
Cannot use with SKR2 mainboard.  Error "pin PA2 is reserved for t5uid1"

PA2 is used for TH0 on this board.

Is there a way to change the default pin??

https://cdn-3d.niceshops.com/upload/file/BIGTREETECH_SKR_2_user_manual.pdf

**Comment by Desuuuu** [comment, marked as answer]
You need to select the interface the touchscreen is connected to when configuring the firmware (enable advanced options if you don't see it).

The TFT port on your board is connected to USART1.

**Reply by phoenixfire425** [reply]
I was able to find that from your source before I had seen your reply!  Is that documented any place?

I was able to set this up with Fluidd and it works great.  Thank you!
